README.md

```
@@ -135,6 +135,53 @@ if (hovered) {
+```jsx
+import Button from './components/Button';
+
+<div className="button-group">
+  <Button variant="primary" href="https://example.com">主要按钮</Button>
+  <Button variant="secondary">次要按钮</Button>
+  <Button variant="ghost" size="sm">幽灵按钮</Button>
+</div>
+```
+
+**Props:**
+| Prop | Type | Default | Description |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)